examples/mixture-fixed-cfl-mpi.py

```diff
@@ -0,0 +1,313 @@
+"""Demonstrate combustive mixture with constant CFL mode."""
```

Comment:
In light of the new example added to `examples`, I was thinking: Do we want to try to cut down the examples and only keep ones that we need for particular functionality? For example, probably don't need `wave-eagar` _and_ `wave-eagar-mpi`? Maybe drop the other `mixture-mpi.py` in favor of this new one? I worry about the steadily increasing CI times, but perhaps It's not that big of a deal?

Reply:
I definitely would like to see the wave stuff go.  It calls very little of the mirgecom infrastructure and I think they are misleading as mirgecom examples. 

Definitely would plan on merging this mixture example to the existing, or just enabling constant CFL mode optionally on all the examples too.

Reply:
I would personally rather see wave-eager or wave-eager-mpi get updated with the modern simutils, etc. stuff when that’s done instead of trashing them. I don’t know that we need to keep both of them though.

Reply:
I'd feel a lot better about the wave examples if they used an operator, and boundary data structures from mirgecom. For me, simutils is a little too driver-adjacent or surface to make the wave examples mirgecom-relevant. They just seem like grudge examples copied to the mirgecom examples dir.

Reply:
Let's add a readme for the examples? I hope this can help us organize our thoughts around what can get the axe.  #392